dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pPanel.vue

```diff
@@ -22,24 +22,17 @@
       <div v-if="shouldCategoryBeRendered(arrayOfKpiDataObjectsMapItem[0])">
         <div>
           <div class="pt-2 pl-2 pb-2 w-full d-cursor-pointer border-bottom-table p-2" @click="toggleExpansion(index)">
-            <div v-if="!isExpanded(index)">
-              <span
-                :class="`p-badge badge-${colorOfCategory(arrayOfKpiDataObjectsMapItem[0])}`"
-                :data-test="arrayOfKpiDataObjectsMapItem[0]"
-                >{{ arrayOfKpiDataObjectsMapItem[0].toUpperCase() }}</span
-              >
-              <button class="pt-1 pr-3 d-cursor-pointer d-chevron-style">
-                <span class="pr-1 pt-1 pi pi-chevron-right d-chevron-font"></span>
-              </button>
-            </div>
-            <div v-if="isExpanded(index)">
-              <span :class="`p-badge badge-${colorOfCategory(arrayOfKpiDataObjectsMapItem[0])}`">{{
-                arrayOfKpiDataObjectsMapItem[0].toUpperCase()
-              }}</span>
-              <button class="pt-2 pr-3 d-cursor-pointer d-chevron-style">
-                <span class="pr-1 pi pi-chevron-down d-chevron-font"></span>
-              </button>
-            </div>
+            <span
+              :class="`p-badge badge-${colorOfCategory(arrayOfKpiDataObjectsMapItem[0])}`"
+              :data-test="arrayOfKpiDataObjectsMapItem[0]"
+              >{{ arrayOfKpiDataObjectsMapItem[0].toUpperCase() }}
+            </span>
+            <button v-if="!isExpanded(index)" class="pt-1 pr-3 d-cursor-pointer d-chevron-style">
+              <span class="pr-1 pt-1 pi pi-chevron-right d-chevron-font"></span>
+            </button>
+            <button v-if="isExpanded(index)" class="pt-2 pr-3 d-cursor-pointer d-chevron-style">
```

Comment:
Just FYI there is also a `v-else`. I'm fine with how it is though.

Reply:
thanks